lib/kafka_ex/server_0_p_10_p_1.ex

```diff
@@ -94,6 +95,15 @@ defmodule KafkaEx.Server0P10P1 do
     {:noreply, update_metadata(state, @metadata_api_version)}
   end
 
+  def kafka_api_versions(state) do
```

Comment:
kafka_api_versions should be defined as a callback in `KafkaEx.Server`

Reply:
You mean I should add this ?

```elixir
  @callback kafka_api_versions(state :: State.t) ::
    {:reply, reply, new_state}
```

If so, should I have not also added one for kafka_create_topics ?

```elixir
  @callback kafka_create_topics(CreateTopicsRequest.t, network_timeout :: integer, state :: State.t) ::
    {:reply, reply, new_state}
```

I can do it here if it's the case.